CONTRIBUTING.md

```diff
@@ -17,6 +17,11 @@ Kubernetes Without Kubelet. Kwok simply simulates the node's behaviour.
 As a result, it can mimic a high number of nodes and pods while consuming
 only a small amount of memory.
 
+Run the following command for a fresh clone to initialize and update the submodule:
```

Comment:
A similar instruction (or an anchor) should be added to the Unit Test section, as the tests will fail without running this command.

Reply:
Done @Omeramsc